solcjs

```diff
@@ -22,6 +22,7 @@ program
   .option('--bin', 'Binary of the contracts in hex.')
   .option('--abi', 'ABI of the contracts.')
   .option('--standard-json', 'Turn on Standard JSON Input / Output mode.')
+  .option('--base-path', 'Use the given path as the root of the source tree instead of the root of the filesystem.')
```

Comment:
Description is ambiguous (what is the root of what?).
Maybe `Specifies the root of the source tree (default value: "/").`?

Reply:
You are right, changed it.